src/pages/using-spark/foundations/typography.mdx

```diff
@@ -263,6 +272,7 @@ Use for all paragraph and non-heading text.
 <ComponentPreview
   componentName="typography--body-two"
   componentType="foundations"
+  title="Body Two Example"
```

Comment:
We should avoid using title as a prop name. I know it doesnt do this for react, but for angular it causes a11y errors. So we'll just want to use something else. ariaTitle, attrTitle, something like that is fine.

Reply:
This has been updated